### PR TITLE
TestCases::pass checks stdout if the file exists

### LINKED ISSUE #64: Check stdout in TestCases::pass
Could `TestCases::pass` with `stdout` do the same as `TestCases::compile_fail` with `stderr` ?

In the event that the file is present, it will be checked.

### PR DESCRIPTION
**Pull request by botika**
Closes #64

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -28,5 +28,8 @@ serde_json = "1.0"
 termcolor = "1.0.4"
 toml = "0.5.2"
 
+[dev-dependencies]
+proc-macro-test = { path = "./proc-macro-test" }
```
Comment (dtolnay) [inline Cargo.toml:32]: This needs to be an actual crate with a version number or else `cargo publish` won't know how to treat it when publishing trybuild. Please change to:

```suggestion
trybuild-test-suite = { version = "0", path = "tests/macro" }
```

File: proc-macro-test/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[package]
+name = "proc-macro-test"
+version = "0.0.1"
+edition = "2018"
+
```
Comment (dtolnay) [inline proc-macro-test/Cargo.toml:5]: Please add `publish = false`.

File: Cargo.toml (hunk 1 above)
Comment (botika) [reply Cargo.toml]: Done

File: proc-macro-test/Cargo.toml (hunk 2 above)
Comment (botika) [reply proc-macro-test/Cargo.toml]: Done

**Comment by botika** [comment]
Done, It should now have the behavior you expect. Thanks for your time!

**Comment by botika** [comment]
It is because when before compiled correctly it has been cached and does not recompile so there is no output.

Should I force recompile in all cases?
How do you want to solve this?

**Comment by botika** [comment]
The problem is when launching the [`cargo clean --package`](https://github.com/dtolnay/trybuild/blob/master/src/cargo.rs#L53-L54). Just clean the crate in question, not the build dependency crates, in this case the `test-suite-trybuild`.
 
Removing these two lines solves the problem, although the compilations will always be complete. Build-dependencies are simple, but I don't know how to detect proc_macro crates,

**Comment by botika** [comment]
If you are testing the same crate it works perfectly. 

Moving these tests to `test-suite-trybuild` crate and warning in the documentation of undesirable caches in dependencies.

**Comment by botika** [comment]
Solved, I move this coverage to `trybuild-test-suite`. But in macos and windows no have stdout, there is something wrong with the macro `print!` in proc-macro context and this systems.

**Comment by botika** [comment]
Closes

**Pull request closed without merging**